BeeSwift/TimerViewController.swift

```diff
@@ -9,18 +9,26 @@
 import UIKit
 import SnapKit
 import MBProgressHUD
+import OSLog
 
 import BeeKit
 
 class TimerViewController: UIViewController {
+    let logger = Logger(subsystem: "com.beeminder.beeminder", category: "TimerViewController")
+
     let timerLabel = BSLabel()
     let startStopButton = BSButton(type: .system)
     let goal: Goal
     var timingSince: Date?
     var timer: Timer?
     var units: String?
     var accumulatedSeconds = 0
-
+    
+    private var idleTimer: Timer?
+    private var defaultBrightnessLevel: CGFloat = UIScreen.main.brightness
```

Comment:
We should probably capture the current value as late as possible (i.e. just before changing it), to minimize surprising behavior if the user changes brightness while on this screen?